docs/conf.py

```diff
@@ -137,20 +142,20 @@
 
 # Add any paths that contain custom themes here, relative to this directory.
 # To use a different custom theme, add the directory containing the theme.
-#html_theme_path = []
+html_theme_path = astropy_sphinx_theme.get_html_theme_path()
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes. To override the custom theme, set this to the
 # name of a builtin theme or the name of a custom theme in html_theme_path.
-#html_theme = None
+html_theme = 'astropy-custom-menu'
```

Comment:
@astrofrog - Can we call it `astropy-coordinated-theme` or something similar? Then all coordinated packages and infrastructure can easily use this theme.

Reply:
Do we actually need to change the name of the theme? Currently it should be ``bootstrap-astropy`` I think - and I think we could just make it an option whether the links are included at the top or not, as opposed to giving the theme a different name.

Reply:
Oh @astrofrog, so I can just go ahead and modify the original theme menubar? But now if I add a second "theme option" people can decide for themselves whether to stick with the old layout or the new layout or opt out, all by changing the `html_theme` from `bootstrap-astropy` to `astropy-custom-menu`... That might be easier for the user to implement and more intuitive for the changes to be understood conceptually.

Reply:
@kakirastern - users are not supposed to use this, in my vision the menubar is for astropy packages only (well, OK, you're right that the users may want to use the same machinery to add a menubar by themselves, but then they probably know what they are doing and not using the astropy theme anyway).

Reply:
Okay @bsipocz, I will make the necessary changes according to your instructions soon. However, I will need some time to figure out how to do it in the way your have envisioned though...

Reply:
Sorry, just picked the way that is the easiest to implement (for me) 😜

Reply:
@kakirastern - just to be clear, I don't think ``html_theme`` should be changed - instead, we should have another configuration item in ``conf.py`` such as e.g. ``astropy_project_menu = True/False`` that controls whether the links are shown

Reply:
Yup @astrofrog, thanks for the tip! You mentioned it before but I wasn't sure how it can be done, but now I guess I have a sense after trying the wrong way first.

Reply:
That way seems really cool, like having an on or off button for the menubar within the "astropy machinery."